### ISSUE TITLE
Essentia Engine compability

### ISSUE THREAD

**Issue by ReTeYeZ** [Lycaonj/AdvancedThaumaturgy#40]
I tried to use the essentia engine with ender io but the conduits don't connect to it.
Does buildcraft has to be installed for this to work?

Anyway if you want compability for other currents i suggest you take a look in the UE api it, afaik, supports MJ, EU, RF and UE joules (obviously).

**Comment by Lycaonj** [comment]
No, you shouldn't NEED Buildcraft.  My engine is coded so that any mod that PROPERLY implements Buildcraft's IPowerEmitter and IPipeConnection should work with the engine.  Now that AT is out of WIP status, I'll be making efforts to properly implement all of the common energy types for compatibility.

**Comment by shadoxxhd** [comment]
for me, the essentia engine doesnt even connect to buildcraft pipes/lasers/energy bridge mj consumers (i tried every side, and have bc installed)
version 0.0.28/minecraft 1.6.4
